jax/_src/numpy/fft.py

```diff
@@ -26,13 +26,18 @@
 from jax._src.numpy import lax_numpy as jnp
 from jax._src.numpy import ufuncs, reductions
 from jax._src.typing import Array, ArrayLike
+from jax._src.numpy.util import promote_dtypes_inexact
```

Comment:
Please add this to the existing imports from `jax._src.numpy.util`, a few lines above.